README.md

```diff
@@ -1,20 +1,171 @@
 ## Overview
 
 ### What is SBOM quality score
+A quality SBOM is one that is accurate, complete, and up-to-date. It should accurately reflect the components and dependencies used in the software application, including their version numbers and optionally any known vulnerabilities. In addition, it should be easily accessible and understandable by stakeholders, such as developers, security teams, and compliance officers.
```

Comment:
1. including their ~version numbers~ versions
2. accessible *to* and understandable by